src/components/RepositoryCard.tsx

```diff
@@ -132,6 +133,8 @@ ref: Ref<any>) {
                     marginLeft={8}
                     paddingHorizontal={8}
                     paddingVertical={8}
+                    backgroundColor="transpanrent"
```

Comment:
There is a typo.

Reply:
Still works ;)

Reply:
I think you can actually replace it with pure `Div` or `Flex` so it will not be needed anymore.

Reply:
The Chip has the size="large" prop and aligns with the Figma file that has such a container.